Web/Admin.UI/src/app/components/tenant/acquisition-log/acquisition-log.service.ts

```
@@ -0,0 +1,310 @@
+    {
+      return this.http.get<AcquisitionLogSummary[]>(`${this.baseUrl}/acquisition-logs?${queryString}`, { headers: headers })
+      .pipe(
+        map((response: AcquisitionLogSummary[]) => {
+          return response;
+        }),
+        catchError((error: HttpErrorResponse) => {
+            var err = this.errorHandler.handleError(error);
+            return err;
+        })
+      )
+    }
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)